src/azure-cli/azure/cli/command_modules/monitor/operations/action_groups.py

```diff
@@ -144,7 +144,8 @@ def _build_arguments_schema(cls, *args, **kwargs):
         args_schema.receiver_actions = AAZCustomListArg(
             options=["--actions"],
             singular_options=["--action", "-a"],
-            help="Add receivers to the action group during the creation." + '''
+            help='''
+        Add receivers to the action group during the creation.
         Usage:   --action TYPE NAME [ARG ...]
```

Comment:
Usage-related message should be set in `long-summary`.

Reply:
I'm not sure what you're asking me to do. All I did was change the concatenation of this multi-line string so it wouldn't be broken for our customers anymore. A change in how these variables are defined would be something out of my scope.